.rubocop.yml

```diff
@@ -1233,6 +1236,7 @@ Rails/SafeNavigation:
   # `try` and `try!` work slighly differently. `try!` and safe navigation will
   # both raise a `NoMethodError` if the receiver of the method call does not
   # implement the intended method. `try` will not raise an exception for this.
+  Enabled: false
```

Comment:
Just found out that there's two `SafeNavigation` cops, `Rails/SafeNavigation` and `Style/SafeNavigation`. Were both causing problems? At first sight, `Rails/SafeNavigation` as we had it configured only affects `try!` so maybe we can keep it?

Reply:
Sounds fair, done!